IBAnimatable/BlurEffectStyle.swift

```diff
@@ -3,10 +3,41 @@
 //  Copyright © 2015 Jake Lin. All rights reserved.
 //
 
-import Foundation
+import UIKit
 
-public enum BlurEffectStyle: String {
-  case extraLight
-  case light
-  case dark
+
+extension UIBlurEffectStyle : IBEnum {
+  /** initialize a UIBlurEffectStyle from string. `extraLight`, `light`, `dark`, `prominent` (iOS 10+), `extraLight` (iOS 10+)
+    default value is `extraLight`.
+   defaultValue is used when the string is nil or not within the above list
+  */
+  public init?(string: String?) {
```

Comment:
Unfortunately it's not applicable in this case. `UIBlurEffectStyle` has rawValue of `Int`, it's the counter part of extending the system `UIBlurEffectStyle`